src/helpers/models.rs

```diff
@@ -0,0 +1,140 @@
+use num_bigint::BigInt;
+use serde::{Deserialize, Serialize};
+use std::fmt::{Debug, Formatter};
+use std::ops::Range;
+
+// Type aliases to indicate whether the parameter should be encrypted, secret shared, etc.
+// Underlying types are temporalily assigned for PoC.
+type CipherText = BigInt;
+type PlainText = String;
+type SecretShare = [CipherText; 3];
+type PBRange = Range<u8>;
```

Comment:
I don't think we want to use a Range for this. A range has a start and an end. The privacy budget a query will consume will be a fixed number, not something within a range.